src/main/react-front-end/src/views/NavbarComponent.js

```diff
@@ -75,8 +88,8 @@ class NavbarComponent extends Component {
 		        {!this.state.login &&
 			        <NavItem id="NavRegister" componentClass={Link} to={registerPageUrl} href={registerPageUrl}>Register</NavItem>
 		    	}
-		        {this.state.login && 
-			        <NavItem id="NavLogout" onClick={()=>{store.dispatch(logoutAction())}}>
+		        {this.state.login &&
+			        <NavItem id="NavLogout" onClick={ this.logout() }>
```

Comment:
Not preferred.
The dispatch action performs multiple operations on logout. removing the action dispatch will not avoid those